### PR TITLE
fix swin compatibility issue with torch 2 and add GPT-3 in homepage

### PR DESCRIPTION
**Pull request by tocean**
#9

### CODE REVIEW

File: Swin-Transformer/Swin-Transformer.patch [hunk 1]
```diff
@@ -84,7 +93,12 @@ index 84230ea..faa9291 100644
  
      args, unparsed = parser.parse_known_args()
  
-@@ -95,9 +106,22 @@ def main(config):
++    args.local_rank = int(os.environ["LOCAL_RANK"])
```
Comment (wkcn) [inline Swin-Transformer/Swin-Transformer.patch:96]: It is more compatible with args.local_rank and env LOCAL_RANK.
```python
    if args.local_rank is None and 'LOCAL_RANK' in os.environ:
        args.local_rank = int(os.environ['LOCAL_RANK'])
```

Comment (tocean) [reply Swin-Transformer/Swin-Transformer.patch]: Fixed.

**Review by wkcn — APPROVED** [review]
LGTM. Thanks!

File: Swin-Transformer/Swin-Transformer.patch [hunk 2]
```diff
@@ -71,7 +71,16 @@ index 84230ea..faa9291 100644
...
-@@ -73,6 +78,12 @@ def parse_option():
+@@ -64,7 +69,7 @@ def parse_option():
+     parser.add_argument('--throughput', action='store_true', help='Test throughput only')
+ 
+     # distributed training
+-    parser.add_argument("--local_rank", type=int, required=True, help='local rank for DistributedDataParallel')
++    parser.add_argument("--local_rank", type=int, help='local rank for DistributedDataParallel')
+ 
+     # for acceleration
+     parser.add_argument('--fused_window_process', action='store_true',
```
Comment (guoshzhao) [inline Swin-Transformer/Swin-Transformer.patch:82]: Why add this line in patch?

Comment (tocean) [reply Swin-Transformer/Swin-Transformer.patch:82]: You mean line 79?  It is not a new line, just remove `required=True`.

Comment (guoshzhao) [reply Swin-Transformer/Swin-Transformer.patch:82]: I mean line 82

Comment (tocean) [reply Swin-Transformer/Swin-Transformer.patch:82]: There is no '+' before line 82, means there is no change to this line. The patch is generated from `git diff`, I guess it adds the context of the changed line.

Comment (guoshzhao) [reply Swin-Transformer/Swin-Transformer.patch:82]: Got it.

**Review by guoshzhao — APPROVED** [review]

**Pull request merged**